src/mongocxx/client.hpp

```diff
@@ -61,7 +68,7 @@ class MONGOCXX_API client {
     ///   Additional options that cannot be specified via the mongodb_uri
     ///
     client(
-        const class uri& mongodb_uri = mongocxx::uri(),
+        const class uri& mongodb_uri,
```

Comment:
I understand the logic behind Databases and Collections having default constructors, but could you explain why this is better for client? It seems weird to make people say `client mongo_client{uri{}};` instead of `client mongo_client{};`.